staging/src/k8s.io/csi-translation-lib/plugins/azure_disk.go

```diff
@@ -208,11 +204,7 @@ func (t *azureDiskCSITranslator) TranslateCSIPVToInTree(pv *v1.PersistentVolume)
 		if fsType, ok := csiSource.VolumeAttributes[azureDiskFSType]; ok && fsType != "" {
 			azureSource.FSType = &fsType
 		}
-
-		if kind, ok := csiSource.VolumeAttributes[azureDiskKind]; ok && kind != "" {
-			diskKind := v1.AzureDataDiskKind(kind)
-			azureSource.Kind = &diskKind
-		}
+		azureSource.Kind = &managed
```

Comment:
> Does this mean if we put lower case intree, it will still translate to upper case?

Yes, here is the key fix. @Jiawei0227 
@msau42 could you approve? thanks.

Reply:
Can we make the CSI driver case insensitive?

Reply:
there is already a fix in CSI driver to make case insensitive